datasets/_analysis/ann_graph_topics/analyzer.py

```diff
@@ -6,57 +8,72 @@
 from zavod.integration import get_dataset_linker
 
 
+def non_graph_topics(context: Context, entity: Entity) -> Set[str]:
+    topic_stmts = entity.get_statements("topics")
+    return {s.value for s in topic_stmts if s.dataset != context.dataset.name}
+
+
+def emit_patch(
+    context: Context,
+    risk_source: Entity,
+    related_entity: Entity,
+    topic: str,
+    existing_topics: Set[str],
+) -> None:
+    context.log.info(
+        f"Adding topic: {topic}",
+        source=risk_source.caption,
```

Comment:
(nit) I think the params have better names than the fields in this log message, why not stick with them? I think they're well-named.